### ISSUE TITLE
Navbar Redesign - Improve Visual Aesthetics and UX for a More Engaging Interface

### ISSUE THREAD

**Issue by Waqar080206** [MrunaliniPachpute/DSA-Patterns-and-Problems#45]
## 🐞 Bug Description
The Current design is Minimal and lacks accessibility.
The interface relies heavily on default or very basic styling.
The lack of visual contrast and modern design elements makes the site appear dated.

---

## 📋 Steps to Reproduce
1. Load Any page 
2.  You will be able to see that Navbar lacks premium ness and its content (Home, About,) lacks proper visibility 

---

## 🎯 Expected Behavior
The Navbar should include:
1. Modernize Styling
2. Typography Improvements:
3. Enhancing

---

## 📸 Screenshots

<img width="1902" height="86" alt="Image" src="https://github.com/user-attachments/assets/ed8b2dae-732e-449c-9a5f-24bf5c83b812" />

---

## 💻 Environment
- OS: [e.g. Windows 11, Ubuntu 22.04]
- Browser: [e.g. Chrome 119, Firefox 120]
- Version: [e.g. v1.0.0]
